ILLinkTasksAssembly is empty blocking C++/CLI

"C:\work\NETCoreCppCliTest\NETCoreCppCliTest.sln" (Restore target) (1) ->
"C:\work\NETCoreCppCliTest\NETCoreCppCliTest\NETCoreCppCliTest.vcxproj" (_IsProjectRestoreSupported target) (2) ->
  C:\Program Files\dotnet\sdk\6.0.100-preview.4.21203.3\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.ILLink.targets(107,32): error MSB4022: The result "" of evaluating the value "$(ILLinkTasksAssem
bly)" of the "AssemblyFile" attribute in element <UsingTask> is not valid. [C:\work\NETCoreCppCliTest\NETCoreCppCliTest\NETCoreCppCliTest.vcxproj]

--- a/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets
+++ b/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.targets
@@ -1074,7 +1074,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.CSharp.targets" Condition="'$(Language)' == 'C#'" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.VisualBasic.targets" Condition="'$(Language)' == 'VB'" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.FSharp.targets" Condition="'$(Language)' == 'F#'" />
-  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.ILLink.targets" />
+  <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.ILLink.targets" Condition="'$(Language)' != 'C++'" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.Sdk.Analyzers.targets" Condition="'$(Language)' == 'C#' or '$(Language)' == 'VB'" />
 
   <!-- Import WindowsDesktop targets if necessary -->

